packages/plugins/Web3Profile/src/SNSAdaptor/utils.ts

```diff
@@ -186,7 +186,7 @@ export const getNFTList = async (walletList: WalletTypes[]) => {
                 return {
                     address,
                     collections: result?.data?.map((asset) => ({
-                        key: `${asset?.contract?.address}+${asset.tokenId}`,
+                        key: `${asset?.contract?.address}_${asset.tokenId}`,
```

Comment:
Any difference?

Reply:
The key of all nft messages in the nft wall is split by _, we need to change it to the same format.